flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/FlinkRelNode.scala

```diff
@@ -101,6 +118,13 @@ trait FlinkRelNode extends RelNode {
     }
   }
 
+  def getExpressionFormat(pw: RelWriter): ExpressionFormat = pw match {
```

Comment:
move this to a utility class? it's weird this interface should know different kinds of `RelWriter`